### PR TITLE
PR-581 Update from upstream v0.15.0 (rc2)

### PR DESCRIPTION
**Pull request by yvon-dblg**
Fix after PR https://github.com/mdblp/shoreline/pull/20 review

See https://diabeloop.atlassian.net/browse/PT-581

`/!\` Still use the go-common in a dev branch (feature/pt582-0.4.1)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,19 +1,17 @@
...
-
-COPY . .
+FROM golang:1.12.7-alpine AS development
 
 RUN apk --no-cache update && \
     apk --no-cache upgrade && \
     apk add build-base git
 
-RUN go get -u github.com/golang/dep/cmd/dep
+# Using Go module (go 1.12 need this variable to be set to enable modules)
```
Comment (CedricPerotto) [inline Dockerfile:8]:
```suggestion
# Using Go module (go 1.12 need this variable to be set to enable modules)
# The variable should default to "on", in Go 1.14 release
```

File: .travis.yml [hunk 2]
```diff
@@ -2,34 +2,16 @@ sudo: false
...
-  - export TRAVIS_BUILD_DIR=$HOME/gopath/src/github.com/tidepool-org/shoreline
-  - cd $HOME/gopath/src/github.com/tidepool-org/shoreline
-  # Download the "dep" binary to bin folder in $GOPATH
-  - curl -L -s https://github.com/golang/dep/releases/download/v${DEP_VERSION}/dep-linux-amd64 -o $GOPATH/bin/dep
-  # Make the binary executable
-  - chmod +x $GOPATH/bin/dep
-
-install:
-  - ./build.sh
+  # Using Go module (go 1.12 need this variable to be set to enable modules)
```
Comment (CedricPerotto) [inline .travis.yml:13]:
```suggestion
  # Using Go module (go 1.12 need this variable to be set to enable modules)
  # The variable should default to "on", in Go 1.14 release
```

File: Dockerfile [hunk 3]
```diff
@@ -29,6 +27,6 @@ WORKDIR /home/tidepool
 
 USER tidepool
 
-COPY --from=development --chown=tidepool /go/src/github.com/tidepool-org/shoreline/dist/shoreline .
+COPY --from=development --chown=tidepool /go/src/github.com/mdblp/shoreline/dist/shoreline .
```
Comment (CedricPerotto) [inline Dockerfile:30]: As a consequence of previous change we should copy it from /go/src/github.com/tidepool-org
```suggestion
COPY --from=development --chown=tidepool /go/src/github.com/tidepool-org/shoreline/dist/shoreline .
```

File: Dockerfile [hunk 4]
```diff
@@ -1,19 +1,17 @@
...
     apk add build-base git
 
-RUN go get -u github.com/golang/dep/cmd/dep
+# Using Go module (go 1.12 need this variable to be set to enable modules)
+ENV GO111MODULE on
 
-RUN dos2unix build.sh && \
-    dos2unix test.sh && \
-    ./build.sh
+WORKDIR /go/src/github.com/mdblp/shoreline
```
Comment (CedricPerotto) [inline Dockerfile:11]: Retested locally the WORKDIR has no influence on the build (we shoreline sources locally and go-common is retrieved from mdblp packages defined in go.mod file) 
```suggestion
WORKDIR /go/src/github.com/tidepool-org/shoreline
```

**Review by CedricPerotto — CHANGES_REQUESTED** [review]
* Comment updates
* WORKDIR changes in Dockerfile are useless
* Respawning comments on log trace  from previous PR

**Comment by yvon-dblg** [comment]
Fix in https://github.com/mdblp/shoreline/pull/22

**Pull request closed without merging**